src/ncp-spinel/SpinelNCPInstance.cpp

```diff
@@ -5784,7 +5785,7 @@ SpinelNCPInstance::handle_ncp_spinel_value_is(spinel_prop_key_t key, const uint8
 			} else if (value == SPINEL_NET_ROLE_LEADER) {
 				update_node_type(LEADER);
 
-			} else if (value == SPINEL_NET_ROLE_DETACHED) {
+			} else if (value == SPINEL_NET_ROLE_DETACHED || value == SPINEL_NET_ROLE_DISABLED) {
```

Comment:
Should update the `spinel.h` file as well?

Reply:
https://github.com/openthread/wpantund/blob/master/third_party/openthread/src/ncp/spinel.h